.buildkite/ml.rayci.yml

```
@@ -257,6 +258,25 @@ steps:
         --skip-ray-installation
     depends_on: [ "mlbuild", "forge" ]
 
+  - label: ":train: ml: gpu doc tests"
+    tags: 
+      - train
+      - tune
+      - doc
+    instance_type: large
```

This should need GPU?
```suggestion
    instance_type: gpu-large
```